host/usb/CMakeLists.txt

```
@@ -28,6 +33,23 @@ if(CONFIG_SOC_USB_OTG_SUPPORTED)
                      )
     list(APPEND include "include")
     list(APPEND priv_includes "private_include")
+
+    if("${IDF_VERSION_MAJOR}.${IDF_VERSION_MINOR}" VERSION_LESS "6.0")
+        # usb_phy driver is present in USB component only in IDF 5.x
+        # If we want to override usb component from esp-idf with managed usb component from esp-usb, we must provide
+        # usb_phy driver for the managed usb component separately by taking src usb_phy.c and
+        # include esp_private/usb_phy.h from the native usb component in esp-idf
+
+        # Get path to usb component inside esp-idf
+        idf_component_get_property(esp_idf_usb_dir usb COMPONENT_OVERRIDEN_DIR)
```

Maybe, the naming would be a bit more helpful when: 
```suggestion
        idf_component_get_property(usb_component_dir usb COMPONENT_OVERRIDEN_DIR)
```